aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -148,96 +188,53 @@ To do so, start by creating a ``cluster.yaml`` with the following content
     name: procluster
     region: us-east-1
     version: 'YOUR_EKS_VERSION'
-    managedNodeGroups:
-    - name: ng-procluster
-    instanceType: t3.small
-    desiredCapacity: 2
-    labels: {role: worker}
-    ami: ami-xxxxx
-    amiFamily: AmazonLinux2
-    ssh:
-        publicKeyName: yoursshkeyname
-    overrideBootstrapCommand: |
-        #!/bin/bash
-        sudo /etc/eks/bootstrap.sh procluster
 
-This config file allows you to create a cluster using the AMI from the previous step,
-with two nodes and SSH access.
 
-Also we use AmazonLinux2 in amiFamily because at this date it's the only native option support by eksctl.
+Add the following content to your file
 
-The "overrideBootstrapCommand" lets you launch the bootstrap script from AWS EKS
-to initialise the nodes. For further cluster customisation see `this <https://eksctl.io/>`_.
 
+.. tabs::
 
-Without FIPS
-^^^^^^^^^^^^
+	.. tab:: With FIPS
 
-Without FIPS enabled, there's no need to reboot the cluster nodes and thus
-the overall process can be simplified by leveraging one of the existing Ubuntu
-EKS AMIs and customising it at deployment time, via cloud-init.
-
-You should use the cloud-init's
-`ubuntu-advantage module <https://cloudinit.readthedocs.io/en/latest/reference/modules.html#ubuntu-advantage>`_.
-For this deployment, you'll also need to have an existing
-`launch template <https://docs.aws.amazon.com/autoscaling/ec2/userguide/launch-templates.html>`_
-on AWS.
-
-launch template user-data
-*************************
-
-On the advanced section of your launch template (user-data section), copy
-the following code (replacing the "token" field by your UA token):
-
-..  code-block:: bash
-
-    MIME-Version: 1.0
-    Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
-
-    --==MYBOUNDARY==
-    Content-Type: text/cloud-config; charset="us-ascii"
-    ubuntu_advantage:
-    token: <ua_contract_token>
-    enable:
-    - esm
-  
-    --==MYBOUNDARY==
-    Content-Type: text/x-shellscript; charset="us-ascii"
+         .. code-block:: yaml
 
-    #!/bin/bash
-    sudo /etc/eks/bootstrap.sh procluster
+            managedNodeGroups:
+            - name: ng-procluster
+            instanceType: t3.small
+            desiredCapacity: 2
+            labels: {role: worker}
+            ami: ami-xxxxx
+            amiFamily: AmazonLinux2
+            ssh:
+                publicKeyName: yoursshkeyname
+            overrideBootstrapCommand: |
+                #!/bin/bash
+                sudo /etc/eks/bootstrap.sh procluster
+            
+         This config file allows you to create a cluster using the AMI from the previous step,
+         with two nodes and SSH access.
 
-    --==MYBOUNDARY==--
-
-Cloud-init will use this user-data to enable ESM on the cluster nodes and bootstrap the AWS EKS cluster.
+         Also we use AmazonLinux2 in amiFamily because at this date it's the only native option support by eksctl.
```

Comment:
Typo/style changes: Change the above sentence to:
"Also, we use AmazonLinux2 as the amiFamily because currently it's the only native option supported by eksctl."